src/drivers/intel/cavs/dmic.c

```diff
@@ -787,41 +773,28 @@ static inline void ipm_helper1(int *ipm, int stereo[], int swap[], int di)
 
 #if DMIC_HW_VERSION == 2
 
-static inline void ipm_helper2(int source[], int *ipm, int stereo[],
-			       int swap[], int di)
+static inline void ipm_helper2(int source[], int *ipm, int di)
 {
 	int pdm[DMIC_HW_CONTROLLERS];
 	int i;
 	int n = 0;
 
+	for (i = 0; i < 4; i++)
```

Comment:
Can we replace 4 with define?

Reply:
OK, it belongs then to dmic.h where the registers details are defined.